src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,34 @@
+.main {
+  display: flex;
+  flex-wrap: wrap;
+
+  @media (min-width : 1024px) {
+    padding: 50px 40px;
+    width: calculate-width();
+    margin: 0 auto;
+  }
+
+  @media (min-width : 1200px) {
```

Comment:
u can do it without media queries just give appropriate paddings for container & min width for content

Reply:
Maybe this variant?